Updates to homepage layout

markbussard opened the pull request:
### Changes made

1. Updates to our app header to include header text and menu icons:
- Decreased font size 
- Added new SVG icon for header logo

2. Centered 'Start Container', i.e, 'Create New Session' and 'Resume My Session' buttons and '25 minutes of productivity' and '5 minutes of relaxing' text and icons.

3. Added more media queries in respect to mobile views to decrease font sizes for buttons, header and settings menu

Closes #137 and #138

alanlwang1 reviewed (commented):
The edit name button disappears on extremely small viewports, is this something we should address? 
![image](https://user-images.githubusercontent.com/34356748/170939154-c7b1a888-34b7-4274-b6b8-ad2b817c2b88.png)
Also the initial landing page text looks off on 1024 px laptop, I think it would be better if all the text was on one line 
![image](https://user-images.githubusercontent.com/34356748/170939568-0d399803-92ef-4480-b360-05a0b2aee5a9.png)

alanlwang1 commented:
Also is it me or does the new logo look fuzzy when compared to the other icons? We might need to increase the resolution of the logo image

alanlwang1 commented:
Should we also get rid of this invisible span? It makes the "5 minutes of relaxing" header look off on small screens as well 
![image](https://user-images.githubusercontent.com/34356748/171478596-103f32dd-918e-4a9e-8573-25a205b75ae3.png)

markbussard commented:
> Also is it me or does the new logo look fuzzy when compared to the other icons? We might need to increase the resolution of the logo image

Addressed and fixed

markbussard commented:
> Should we also get rid of this invisible span? It makes the "5 minutes of relaxing" header look off on small screens as well ![image](https://user-images.githubusercontent.com/34356748/171478596-103f32dd-918e-4a9e-8573-25a205b75ae3.png)

Addressed and fixed!

LJMNZS reviewed (changes requested):
![image](https://user-images.githubusercontent.com/47438076/171502983-b9b3bea5-b148-4f4e-a7b1-1bebe4b53ee6.png)
The home page seems to be loading in the top left of the timer page where the logo should be

alanlwang1 commented:
> The edit name button disappears on extremely small viewports, is this something we should address? ![image](https://user-images.githubusercontent.com/34356748/170939154-c7b1a888-34b7-4274-b6b8-ad2b817c2b88.png) Also the initial landing page text looks off on 1024 px laptop, I think it would be better if all the text was on one line ![image](https://user-images.githubusercontent.com/34356748/170939568-0d399803-92ef-4480-b360-05a0b2aee5a9.png)

I'm still seeing these issues. I think the title header font size or the spacing between title/icons needs to be tweaked slightly

markbussard commented:
@alanlwang1 Can you include the viewport size you are using for the mobile issue.

alanlwang1 commented:
> @alanlwang1 Can you include the viewport size you are using for the mobile issue.

I was just using the 'Mobile S' viewport in inspect element, which has a width of 320 px

alanlwang1 commented:
Should we adjust the spacing of the title + menu icons so that all 4 menu icons can fit?
The app page on the same small size looks off since the homepage only needs to care about 3 icons 
![image](https://user-images.githubusercontent.com/34356748/171512282-6c2af368-a69b-4d4a-9b98-860758da4832.png)

markbussard commented:
> Should we adjust the spacing of the title + menu icons so that all 4 menu icons can fit? The app page on the same small size looks off since the homepage only needs to care about 3 icons ![image](https://user-images.githubusercontent.com/34356748/171512282-6c2af368-a69b-4d4a-9b98-860758da4832.png)

Let's make a separate issue for this.
